contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -212,10 +214,18 @@ public VoldemortBuildAndPushJob(String name, azkaban.utils.Props azkabanProps) {
                 if (clusterURLs.contains(url)) {
                     throw new VoldemortException("the URL: " + url + " is repeated in the "+ PUSH_CLUSTER + " property ");
                 }
-                this.clusterURLs.add(url);
-                AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
-                this.adminClientPerCluster.put(url, adminClient);
-                this.closeables.add(adminClient);
+                try {
+                    AdminClient adminClient = createAdminClient(url, fetchAllStoresXml);
+                    this.clusterURLs.add(url);
+                    this.adminClientPerCluster.put(url, adminClient);
+                    this.closeables.add(adminClient);
+                } catch (Exception e) {
```

Comment:
Yeah, but that was the default behavior up to some time ago. I'm not sure when we introduced this change in semantics (I could try to find out, if you're curious...).

Are you suggesting that we should fail the job at the end of the `run()` if some cluster was unreachable during initialization?